capstone-app/src/app/modules/photos/services/photo-fetcher.service.ts

```diff
@@ -1,28 +1,35 @@
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
-import { Observable } from 'rxjs';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { Observable, throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
 
 @Injectable({
   providedIn: 'root',
 })
 export class PhotoFetcher {
-
   private static LINK = 'https://www.googleapis.com/customsearch/v1?';
   private static API_KEY = 'key=AIzaSyDI9uo_xPRWlgF9bSWnJl1xSbeb394tgTo';
   private static SEARCH_ENGINE = 'cx=014012661603529902036:grbt2ttpety';
   private static SEARCH_TYPE = 'searchType=image';
-  private query = 'q=phoenixville';
-  private num = 'num=10';
 
-  constructor(private readonly http: HttpClient) {}
+  constructor(private readonly http: HttpClient) { }
+
+  getPhotos(query: string, limit: number): Observable<any> {
```

Comment:
I am unsure that Observable<any> is the best type here. I think we should create an interface for the service that contains the expected fields and do an Observable of that